src/main/kotlin/org/team_leg3nd/tarae/service/ExpenseService.kt

```diff
@@ -0,0 +1,43 @@
+package org.team_leg3nd.tarae.service
+
+import org.springframework.stereotype.Service
+import org.team_leg3nd.tarae.entity.Expense
+import org.team_leg3nd.tarae.exception.*
+import org.team_leg3nd.tarae.repository.ExpenseRepository
+
+@Service
+class ExpenseService(
+    private val expenseRepository: ExpenseRepository,
+) {
+
+    // 지출 생성
+    fun createExpense(expense: Expense): Expense {
+        return expenseRepository.save(expense)
+    }
+
+    // 지출 조회
+    fun getExpense(id: String): Expense {
+        return expenseRepository.findById(id).orElseThrow { ExpenseNotFoundException("Expense not found") }
```

Comment:
혹시 `findByIdOrNull(id)` 함수는 없었을까요? 있으면 다음과 같이도 표현할 수 있어서요.
```suggestion
        return expenseRepository.findByIdOrNull(id) ?: throw ExpenseNotFoundException("Expense not found")
```

Reply:
오 적용해볼게요